src/genie/libs/parser/iosxe/show_license.py

```diff
@@ -28,15 +28,15 @@ class ShowLicenseSchema(MetaParser):
         'licenses': {
             int: {
                 'feature': str,
-                'period_left': str,
+                Optional('period_left'): str,
```

Comment:
could you please add unittest for the output which we need to make these as `Optional`?

Reply:
added unittest with items needed optional